src/components/Checkout/CheckoutAPICalls.tsx

```diff
@@ -49,4 +47,73 @@ export async function getRecentTransactions(buildingCode: string, unitNumber: st
     })
   });
   return await response.json();
+}
+
+export async function getUnitNumbers(buildingId: number) {
+  // returns a list of unit codes for a given building code
```

Comment:
I am not sure what this comment adds to the code? It is clear to me what it does. I would remove it.